state/src/impls/top_level.rs

```diff
@@ -490,6 +536,24 @@ impl TopLevelState {
         Ok(())
     }
 
+    fn create_module_level_state(&mut self, storage_id: StorageId) -> StateResult<()> {
+        const DEFAULT_MODULE_ROOT: H256 = BLAKE_NULL_RLP;
+        {
+            let module_cache = self.module_caches.entry(storage_id).or_default();
+            ModuleLevelState::from_existing(storage_id, &mut self.db, DEFAULT_MODULE_ROOT, module_cache)?;
+        }
+        ctrace!(STATE, "module storage({}) created", storage_id);
+        self.set_module_root(storage_id, DEFAULT_MODULE_ROOT)?;
+        Ok(())
+    }
+
+    #[allow(dead_code)]
```

Comment:
@HoOngEe  I added this annotation to your changes since it's not used yet.

Reply:
Ok, thank you.